### DISCUSSION TITLE
Welcome to itnconverter Discussions!

### DISCUSSION THREAD

**Discussion by Benichou34** [Benichou34/itnconverter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Quico1979** [comment]
Hi. In trip planner, when I search for a place or city, I keep getting the message "the geographic location cannot be returned due to legal or contractual reasons" What am I missing here? Regards.

**Comment by CX72TMZ** [comment]
For several years I have been a very satisfied and grateful user of Benichou Software's ITNConverter. I have been away from my computer without the need for the programme for a few weeks and now find my saved routes cannot be loaded or modified and it is not possible to plan a new itinerary. I am not at all technical and find much of the content of GitHub way beyond my understanding but hope there may be a simple solution to be able to restore or renew my access. I very much hope you can help. Typed with my fingers crossed! SF

**Comment by Bikereddu** [comment]
Fantastico, ti ho ritrovato :) il mio miglior editor di giri.
